lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -368,7 +368,7 @@ class MongoClientInterface {
      * to manage the case of an object created before the
      * versioning was enabled.
      */
-    putObjectVerCase1(c, bucketName, objName, objVal, params, log, cb) {
+    putObjectVerCase1(c, bucketName, objName, objVal, params, log, cb, isRetry) {
```

Comment:
not a fan of adding a parameter after the callback one, but it would probably break a lot of stuff if we swapped them. hmm

Reply:
yes me neither, but i was focusing on fixing stuff :)